inst/include/frogger.hpp

```diff
@@ -3,37 +3,43 @@
 #include "general_demographic_projection.hpp"
 #include "hiv_demographic_projection.hpp"
 
-template <typename real_type>
-State<real_type> run_model(int time_steps, const Parameters<real_type>& pars) {
-  State<real_type> state(pars.age_groups_pop, pars.num_genders,
-                      pars.disease_stages, pars.age_groups_hiv,
-                      pars.treatment_stages);
+namespace leapfrog {
```

Comment:
prevailing style is generally not to indent namespaces, you should be able to configure that in your editor

Reply:
Ok, I've updated those rules and run the autoformatted. Has updated some other lines like moving the reference from `const Parameters<real_type>& pars` to `const Parameters<real_type> &pars`